README.md

```diff
@@ -130,6 +130,16 @@ Note that a *graph ID* is an ID found in BGV files, but is not unique. A
 BGV 7.0
 ```
 
+To simplify automation use cases, you can also capture the output as JSON:
+
+```
+% seafoam --json examples/fib-java.bgv.gz info | jq --sort-keys
+{
+  "major_version": 7,
```

Comment:
@chrisseaton Should this output have something like `"type": "BGV"` to match the plain-text response? Or is BGV essentially implied?